client/src/pages/Login/DemoLogin/DemoLogin.tsx

```diff
@@ -0,0 +1,55 @@
+import Button from '@mui/material/Button';
+import Box from '@mui/material/Box';
+import CircularProgress from '@mui/material/CircularProgress';
+import { Formik, FormikHelpers } from 'formik';
+import useStyles from './useStyles';
+
+interface Props {
+  handleSubmit: (
+    {
+      email,
+      password,
+    }: {
+      email: string;
+      password: string;
+    },
```

Comment:
Refactoring this type into a small interface would clean up most of this code 👍🏼